### PR TITLE
Ap/feature/responsive design

### PR DESCRIPTION
**Pull request by Adelle-Pitsas**
#### What does this PR do?
- Adds responsive styling
- Adds loaders for pages to replace "loading..."

#### Where should your reviewer start
- All view pages that contain other components

#### What (if any) features are you implementing
- responsive design
- loaders

#### Is there anything that you need from your teammate?
pull ye ol` branch down and change ye' ol' dimensions to test ye ol' app

### CODE REVIEW

File: src/Components/MyBooks/MyBooks.css [hunk 1]
```diff
@@ -18,6 +18,15 @@
...
+}
+
+@media screen and (max-width: 920px) {
+  .my-books-container {
+    display: flex;
+    flex-direction: column;
+    width: fit-content;
+    align-items: center;
+  }
+
```
Comment (FrazierLE) [inline src/Components/MyBooks/MyBooks.css:30]: @726px, it should stack over each other like it does on the home page so it avoids the break

Comment (FrazierLE) [reply src/Components/MyBooks/MyBooks.css:30]: Screenshot 2023-02-15 at 5 24 53 PM

Comment (FrazierLE) [reply src/Components/MyBooks/MyBooks.css:30]: potentially a flex-wrap: wrap

File: src/Components/BorrowedBooks/BorrowedBooks.css [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+
+@media screen and (max-width: 920px) {
+  .borrowed-book-section{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: fit-content;
+    align-items: center;
+  }
+
```
Comment (FrazierLE) [inline src/Components/BorrowedBooks/BorrowedBooks.css:25]: Breaks @732px

File: src/Components/AddBook/AddBook.css [hunk 3]
```diff
@@ -69,4 +68,42 @@
...
+  .search-button {
+    font-size: 16px;
+    margin-left: 0;
+    margin-top: 5px;
+  }
+
+  .search-container {
+    height: fit-content;
+  }
 }
```
Comment (FrazierLE) [inline src/Components/AddBook/AddBook.css:129]: Screenshot 2023-02-15 at 5 27 42 PM
Breaks right after @520px

Comment (FrazierLE) [reply src/Components/AddBook/AddBook.css:129]: Resolved!

File: src/Components/BorrowedBooks/BorrowedBooks.css (hunk 2 above)
Comment (FrazierLE) [reply src/Components/BorrowedBooks/BorrowedBooks.css:25]: Okay I don't know if you just fixed it, but it's all good now

File: src/Components/MyBooks/MyBooks.css (hunk 1 above)
Comment (FrazierLE) [reply src/Components/MyBooks/MyBooks.css:30]: Fixed!

Comment (Adelle-Pitsas) [reply src/Components/MyBooks/MyBooks.css:30]: Wait this is so weird... that's not the design that I have on my branch

**Pull request merged**